internal/envoy/bootstrap.go

```diff
@@ -84,6 +84,9 @@ type BootstrapConfig struct {
 	// referenced in the configuration actually exist. This option is for
 	// testing only.
 	SkipFilePathCheck bool
+
+	// DNSLookupFamily
```

Comment:
Should we add a descriptive comment here? At least to make the linter etc. happy (though it is a bit redundant and an internal struct field)

Reply:
Yes, we absolutely should have a more descriptive comment here. Better to have too much documentation than not enough.

Reply:
done